R/data_rename.R

```diff
@@ -257,7 +266,14 @@ data_rename <- function(data,
       # if so, iterate all tokens
       for (token in matches) {
         # evaluate token-object from the environment
-        values <- .dynEval(str2lang(gsub("\\{(.*)\\}", "\\1", token)))
+         values <- tryCatch(
+          .dynEval(str2lang(gsub("\\{(.*)\\}", "\\1", token))),
+          error = function(e) {
+            insight::format_error(paste0(
+              "The object `", token, "` was not found. Please check if it really exists."
+            ))
+          }
+        )
```

Comment:
`.dynEval()` has an argument `ifnotfound` that you can use instead of `tryCatch()`.